scripts/zones/Western_Altepa_Desert/Zone.lua

```diff
@@ -17,6 +18,12 @@ end
 function onInitialize(zone)
     UpdateNMSpawnPoint(ID.mob.KING_VINEGARROON)
     GetMobByID(ID.mob.KING_VINEGARROON):setRespawnTime(math.random(900, 10800))
+
+    dsp.bmt.updatePeddlestox(dsp.zone.WESTERN_ALTEPA_DESERT)
```

Comment:
Just pass along the `zone` parameter the function has already.

Reply:
Resolved in https://github.com/DarkstarProject/darkstar/pull/6095/commits/7e36af9746c302b69aef5cb21bd98209fd7e6e59